ndk-sys/generate_bindings.sh

```diff
@@ -1,6 +1,16 @@
 #!/bin/sh
 
-sysroot="${ANDROID_NDK_ROOT}"/toolchains/llvm/prebuilt/linux-x86_64/sysroot/
+os=$(uname -s)
+
+if [[ "$os" == "Darwin" ]]; then
+    host_tag="darwin-x86_64"
+elif [[ "$os" == "CYGWIN"* ]]; then
+    host_tag="windows-x86_64"
+else
+    host_tag="linux-x86_64"
+fi
+
+sysroot="${ANDROID_NDK_ROOT}"/toolchains/llvm/prebuilt/${host_tag}/sysroot/
```

Comment:
I'll probably push a regen to this PR later before merging it, since while it seems this unblocks you from writing the initial bindings a different host OS causes a lot of unnecessary code-motion and bloated diff in the generated files (even if they may in the end compile to roughly? the same crate).

Reply:
Done. Unfortunately thanks to the way `wrapper.h` is ordered this adds some new `_bindgen_ty_xxx` which bump the numbers everywhere else in this file :(